From 309ce8c2537ae72e2c7a07b7e21bee5fd2da0960 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sat, 1 Nov 2014 13:52:14 -0400
Subject: [PATCH] systemd-journal-flush.service: remove "trigger" from
 description

This service is now synchronous, so "trigger" is misleading.

(cherry picked from commit a65b82457735df2ef58736a55846f400124a8dc0)
---
 NEWS                                   | 7 ++++---
 units/systemd-journal-flush.service.in | 2 +-
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/NEWS b/NEWS
index afc73a781f..ac987ad5b7 100644
--- a/NEWS
+++ b/NEWS
@@ -7,9 +7,10 @@ CHANGES WITH 217:
           show log timestamps in the UTC timezone. journalctl now also
           accepts -n/--lines=all to disable line capping in a pager.
 
-        * journalctl gained a new switch, --flush, that flushes
-          logs from /run/log/journal to /var/log/journal if
-          persistent storage is enabled.
+        * journalctl gained a new switch, --flush, that synchronously
+          flushes logs from /run/log/journal to /var/log/journal if
+          persistent storage is enabled. systemd-journal-flush.service
+          now waits until the operation is complete.
 
         * Services can notify the manager before they start a reload
           (by sending RELOADING=1) or shutdown (by sending
diff --git a/units/systemd-journal-flush.service.in b/units/systemd-journal-flush.service.in
index 2612220e2e..fa290897d6 100644
--- a/units/systemd-journal-flush.service.in
+++ b/units/systemd-journal-flush.service.in
@@ -6,7 +6,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Trigger Flushing of Journal to Persistent Storage
+Description=Flush Journal to Persistent Storage
 Documentation=man:systemd-journald.service(8) man:journald.conf(5)
 DefaultDependencies=no
 Requires=systemd-journald.service
